### PR TITLE
<fix> windows arm PowerShell select to x86_64 exe binary

### PR DESCRIPTION
**Pull request by lucloner**
### Prerequisites

- [x] I have read and understand the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### Description

fix:
just compile arm build and repair if statement.
tested in win10 arm 19042.868 PowerShell arm 7.0.3
binary build go 1.16.2 windows/arm

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: packages/powershell/oh-my-posh/oh-my-posh.psm1 [hunk 1]
```diff
@@ -15,7 +15,10 @@ function Get-PoshCommand {
         }
         return "$PSScriptRoot/bin/posh-linux-amd64"
     }
-    if ([Environment]::Is64BitOperatingSystem) {
+    if ((Get-CimInstance Win32_OperatingSystem).OSArchitecture -icontains "arm") {
```
Comment (JanDeDobbeleer) [inline packages/powershell/oh-my-posh/oh-my-posh.psm1:18]: This fails on Unix and macOS

File: packages/powershell/oh-my-posh/oh-my-posh.psm1 [hunk 2]
```diff
@@ -15,7 +15,10 @@ function Get-PoshCommand {
...
+        return "$PSScriptRoot/bin/posh-windows-arm.exe"
```
Comment (JanDeDobbeleer) [inline packages/powershell/oh-my-posh/oh-my-posh.psm1:19]: this binary isn't part of the module. In fact, it's not even working atm. In theory, the latest insider build should allow running `posh-windows-amd64.exe`

Comment (lucloner) [reply packages/powershell/oh-my-posh/oh-my-posh.psm1:19]: OK. I just want to help. Follow the Tutorial: Set up Powerline in Windows Terminal would receive errors in Windows Arm.
I tried x86 build. It is very slow in git directory (and git is also 32-bit). Then I build arm binary for better performance. But its just arm 32-bit. Golang only support armv7 in Windows.

Comment (JanDeDobbeleer) [reply packages/powershell/oh-my-posh/oh-my-posh.psm1:19]: I've been trying to get it to cross compile but Windows isn't supporting ARM properly (you can't even install it on the latest raspberry PI's to setup a build system). I want to support it, but Windows is giving me a hard time. So I'm afraid I can't accept this until we can also create this binary during our CD pipeline.

**Comment by JanDeDobbeleer** [comment]
Context: until we have a build environment to create valid Windows ARM binary, this can't be supported. See #461.

**Pull request closed without merging**